src/cbc_sdk/platform/users.py

```diff
@@ -0,0 +1,342 @@
+        raise ServerError(500, f"registration return was unsuccessful: {resp['registration_type']}")
+
+    def _refresh(self):
+        """
+        Rereads the user data from the server.
+
+        Returns:
+            bool: True if refresh was successful, False if not.
+        """
+        userdata = self._cb.get_object(self.urlobject.format(self._cb.credentials.org_key))
+        rawdata = [user for user in userdata.get('users', []) if user.get('login_id', 0) == self._model_unique_id]
+        if len(rawdata) == 0:
+            return False
+        self._info = rawdata[0]
+        self._full_init = True
+        self._last_refresh_time = time.time()
+        return True
+
+    def _update_object(self):
+        """
+        Updates the user data on the server.
+
+        Returns:
+            int: The user ID for this user.
+        """
+        if 'login_id' not in self._info:
+            raise ApiError("user should have already been created")
+        url = self.urlobject_single.format(self._cb.credentials.org_key, self._model_unique_id)
+        self._cb.put_object(url, self._info)
+        return self._model_unique_id
+
+    def _delete_object(self):
+        """Deletes the user."""
+        url = self.urlobject_single.format(self._cb.credentials.org_key, self._model_unique_id)
+        self._cb.delete_object(url)
+        return self._model_unique_id
+
+    @property
+    def urn(self):
+        """
+        Returns the URN for this user (used in accessing Grants).
+
+        Returns:
+            str: URN for this user.
+        """
+        return f"psc:user:{self._cb.credentials.org_key}:{self._info['login_id']}"
+
+    @classmethod
+    def create(cls, cb, template=None):
+        """
+        Creates a new user.
+
+        Args:
+            cb (CBCloudAPI): A reference to the CBCloudAPI object.
+            template (dict): Optional template data for creating the new user.
+
+        Returns:
+            User: New user created, if template is specified.  First element in returned tuple.
+            str: Temporary password for new user, if template is specified. Second element in returned tuple.
+            UserBuilder: If template is None, returns an instance of this object. Call methods on the object to set
+                         the values associated with the new user, and then call build() to create it.
+        """
+        if template:
+            my_templ = copy.deepcopy(template)
+            my_templ['org_id'] = 0
+            my_templ['role'] = 'DEPRECATED'
+            if 'auth_method' not in my_templ:
+                my_templ['auth_method'] = 'PASSWORD'
+            return User._create_user(cb, my_templ)
+        return User.UserBuilder(cb)
+
+    def reset_google_authenticator_registration(self):
+        """Forces Google Authenticator registration to be reset for this user."""
+        url = self.urlobject_single.format(self._cb.credentials.org_key, self._model_unique_id) + "/googleauth"
```

Comment:
We received an updated swagger, there was a mistake in the original. The URL for the reset is `/google-auth`, instead of `/googleauth`

Reply:
Got that changed, thanks!